iceoryx_posh/include/iceoryx_posh/capro/service_description.hpp

```diff
@@ -68,7 +68,7 @@ constexpr const char* INTERFACE_NAMES[] = {"INTERNAL", "ESOC", "SOMEIP", "AMQP",
 enum class Scope : uint16_t
 {
     WORLDWIDE,
-    INTERNAL,
+    INTERNAL, /// @todo rename this to SINGLE_MACHINE?
     INVALID
```

Comment:
Nope, this was was a "note to self". Just missed a good name, `LOCAL` is perfect. Fixed.

Reply:
Fast as food for thought. Since you changed `INTERNAL`, what do you think of also changing `WORLDWIDE` to e.g. `GLOBAL`. The `WORLDWIDE` always bugged me and we even don't use it in our codebase except in `ScopeTypeString` and in a test

Reply:
As this is mainly meant to be for gateways to avoid short-circuting, I would prefer `WORLDWIDE` instead of `GLOBAL`. For me it's better understandable. What does global in the end mean? It could end in your car, but maybe also be applicable for a country or a region? Worldwide is more explicit.

Reply:
Well, we don't use it anywhere so it could also just be removed 😅
I also wouldn't connect `WORLD_WIDE` with gateway short-circuiting. Can you tell me more of the exact semantic of this flag?

Reply:
@elBoberido I might have been too tired yesterday eve. You're right, it has nothing to do with shirt-circuiting :sweat_smile:  Nevertheless it's an important property of the `ServiceDescription`. Based on this flag a gateway can decide how far a `ServiceDescription` is propagated. Different reaches could be e.g. one ECU in a car, all ECU inside a car or also the cloud infrastructure which all cars are embedded into. I guess we'll tackle and re-question it anyway with #1141. If we don't find it useful for the DDS gateway, I'm ok to remove it. Nothing for `v2.0`.